### PR TITLE
Modify clown and mime sexy masks and add them to the autodrobe

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Someone requested this to be done and was simple enough.

## Why / Balance
More content + less EMAG locked clothing.

## Technical details

YAML changes.

## Media
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Hopefully nothing.

**Changelog**

:cl:
- tweak: The blushing mime mask and the purple clown mask can now be obtained at an autodrobe.

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml [hunk 1]
```diff
@@ -100,11 +100,17 @@
     ClothingShoesBootsCowboyWhite: 1
     ClothingHeadTechPriest: 2 #Delta V: Added Techpriest Robe to Inventory
     ClothingOuterRobeTechPriest: 2
+# Begin DeltaV  modifications - Moved to regular inventory.
+    ClothingMaskSexyClown: 1
+    ClothingMaskSexyMime: 1
+# End of DeltaV modifications.
```
Comment (deltanedas) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml]: for these have the comments indent at the same level

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Comment by Avalon-Proto** [comment]
I feel like I should mention the fact the samurai dogi is actually decent armor

**Comment by deltanedas** [comment]
this pr doesnt touch dogi

**Comment by Avalon-Proto** [comment]
... I'm stupid I didn't see the Emag by the 2nd machine

**Review by Lyndomen — APPROVED** [review]

**Pull request merged**